Platforms/Graphics/.BlazorGL/Shader/ConcreteShader.cs

```diff
@@ -51,6 +59,44 @@ internal void CreateShader(GraphicsContextStrategy contextStrategy, WebGLShaderT
             }
         }
 
+        static Regex rgxOES = new Regex(
+                @"#extension GL_OES_standard_derivatives : enable", System.Text.RegularExpressions.RegexOptions.Multiline);
+        static Regex rgxAttribute = new Regex(
+            @"^attribute(?=\s)", System.Text.RegularExpressions.RegexOptions.Multiline);
+        static Regex rgxVarying = new Regex(
+            @"^varying(?=\s)", System.Text.RegularExpressions.RegexOptions.Multiline);
+        static Regex rgxFragColor = new Regex(
+            @"^#define ps_oC0 gl_FragColor", System.Text.RegularExpressions.RegexOptions.Multiline);
```

Comment:
I'm not 100% sure that the variable name will always be `ps_oC0`. Maybe this variable name should be extracted for the replacement?

Reply:
good point. Maybe it's different when attaching multiple rendertargets.